### PR TITLE
ceph-volume docs

### PR DESCRIPTION
**Pull request by alfredodeza**
These are roughly the same as what is currently at http://docs.ceph.com/ceph-volume/master/

It was adapted to fit a bit better for the Ceph docs.

### CODE REVIEW

File: doc/ceph-volume/lvm/activate.rst [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+     /var/lib/ceph/osd/<cluster name>-<osd id>
+
+* Preferably, no other mechanisms to mount the volume should exist
+* There is currently no support for encrypted volumes
+
+The one time *activation* process for an existing OSD, with an ID of 0 and
+using a `"ceph"` cluster name would look like::
+
+    ceph-lvm activate /var/lib/ceph/osd/ceph-0
```
Comment (andrewschoen) [inline doc/ceph-volume/lvm/activate.rst]: Is this functionality supported in ``ceph-volume`` yet? I didn't think it was. If not, we should remove this section or mark it so readers will know it's not implemented yet.

File: doc/ceph-volume/lvm/activate.rst [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+# ensure that all required devices are ready for that OSD
+
+# start the ``ceph-osd@0`` systemd unit
+
+
+Summary
+-------
+To recap the ``activate`` process:
+
+#. require both :term:`OSD id` and :term:`OSD uuid` or a mounted path for
```
Comment (andrewschoen) [inline doc/ceph-volume/lvm/activate.rst]: does ``ceph-volume lvm activate`` really work with a mounted path?

File: doc/ceph-volume/lvm/prepare.rst [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+* ``--bluestore``
+
+.. when available, this will need to be updated to:
+.. * :ref:`--bluestore <ceph-volume-prepare_bluestore>`
+
+.. _ceph-volume-lvm-prepare_filestore:
+
+`filestore`
+-----------
+This is the default OSD backend and llows preparation of logical volumes for
```
Comment (andrewschoen) [inline doc/ceph-volume/lvm/prepare.rst]: s/and llows/and allows

File: doc/ceph-volume/lvm/prepare.rst [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+a :term:`filestore` OSD.
+
+The process is *very* strict, for the first initial releases of ``ceph-lvm``.
+It requires two logical volumes that are ready to be used. No special
+preparation is needed for these volumes other than following the minimum size
+requirements for data and journal.
+
+The API call looks like::
+
+    ceph-lvm prepare filestore --data data --journal journal
```
Comment (andrewschoen) [inline doc/ceph-volume/lvm/prepare.rst]: should be ``--filestore``

File: doc/ceph-volume/lvm/prepare.rst [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+     /var/lib/ceph/osd/<cluster name>-<osd id>
+
+* Preferably, no other mechanisms to mount the volume should exist, and should
+  be removed (like fstab mount points)
+* There is currently no support for encrypted volumes
+
+The one time process for an existing OSD, with an ID of 0 and
+using a ``"ceph"`` cluster name would look like::
+
+    ceph-volume lvm prepare --filestore --osd-id 0 --osd-fsid E3D291C1-E7BF-4984-9794-B60D9FA139CB
```
Comment (andrewschoen) [inline doc/ceph-volume/lvm/prepare.rst]: Should we include a note here on how to find the ``osd-fsid`` of an existing OSD?

File: doc/ceph-volume/lvm/prepare.rst [hunk 6]
```diff
@@ -0,0 +1,142 @@
...
+can later be startednot contact the monitor to generate an OSD ID and will
+format the LVM device in addition to storing the metadata on it so that it can
+later be started (for detailed metadata description see :ref:`ceph-volume-lvm-tags`).
+
+
+.. _ceph-volume-lvm-prepare_bluestore:
+
+`bluestore`
+-----------
+This subcommand is planed but not currently implemented.
```
Comment (andrewschoen) [inline doc/ceph-volume/lvm/prepare.rst]: s/planed/planned

File: doc/ceph-volume/lvm/prepare.rst (hunk 5 above)
Comment (alfredodeza) [inline doc/ceph-volume/lvm/prepare.rst]: can you believe I can't find anywhere where this is documented? I feel like this should be a separate PR that describes the contents of the OSD data directory

**Pull request merged**